lib/seo_meta.rb

```diff
@@ -34,8 +34,13 @@ def is_seo_meta(options = {})
     end
 
     # Let SeoMetum know about the base
-    ::SeoMetum.send :belongs_to, self.name.underscore.gsub('/', '_').to_sym,
-                    :class_name => self.name
+    if ActiveRecord::VERSION::STRING >= '5.0.0'
+      ::SeoMetum.send :belongs_to, self.name.underscore.gsub('/', '_').to_sym,
+                      class_name: self.name, optional: true
```

Comment:
wWhat happens if we pass this key to Rails versions less than 5? What I'm asking is do we need the conditional code?